### PR TITLE
[Deploment] Add DPM service URL in NMM yaml file

### PR DESCRIPTION
**Pull request by pkommoju**
The service URL was missing in node manager's kubernetes configuration. This PR adds the two missing URL.

### CODE REVIEW

**Comment by cj-chung** [comment]
LGTM

File: kubernetes/services/node_manager.yaml [hunk 1]
```diff
@@ -18,6 +18,8 @@ data:
     spring.servlet.multipart.file-size-threshold=2KB
     spring.servlet.multipart.max-file-size=200MB
     spring.servlet.multipart.max-request-size=215MB
+    microservices.dataplane.service.url=http://dataplanemanager-service.default.svc.cluster.local:9010/network-configuration
```
Comment (xieus) [inline kubernetes/services/node_manager.yaml:21]: Please check PM's yaml file. The port number should be adjusted to 300XX for K8s deployment.

File: kubernetes/services/node_manager.yaml [hunk 2]
```diff
@@ -18,6 +18,8 @@ data:
     spring.servlet.multipart.file-size-threshold=2KB
     spring.servlet.multipart.max-file-size=200MB
     spring.servlet.multipart.max-request-size=215MB
+    microservices.dataplane.service.url=http://localhost:9010/network-configuration
+    microservices.dataplane.nodeservice.url=http://localhost:9010/nodes
```
[the remark this answers is not in the document]
Comment (pkommoju) [reply kubernetes/services/node_manager.yaml]: One more bug. I think there must be an automated method to do these kind of plumbing changes. I will think about something, meanwhile I will fix it right away.

File: kubernetes/services/node_manager.yaml (hunk 1 above)
Comment (pkommoju) [reply kubernetes/services/node_manager.yaml]: Fixed and updated PR.

Comment (cj-chung) [reply kubernetes/services/node_manager.yaml]: I think either 90xx or 300xx are both ok here, but 300xx will pass though API gw.

File: kubernetes/services/node_manager.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (pkommoju) [reply kubernetes/services/node_manager.yaml]: 90xx is what I saw in other yaml files.

File: kubernetes/services/node_manager.yaml (hunk 1 above)
Comment (xieus) [reply kubernetes/services/node_manager.yaml]: My bad. 90xx works here.

**Pull request merged**